cli/cmd/cluster-apply.go

```diff
@@ -88,6 +89,18 @@ func runClusterApply(cmd *cobra.Command, args []string) {
 		ctxLogger.Fatalf("Verify cluster: %v", err)
 	}
 
+	// Update kube-system namespace with labels.
```

Comment:
I don't think this will work with the reasons mentioned earlier. I think we also need to provide the manifests and patch functionality to bootkube to include all cases, as I mentioned earlier.

Alternatively, we could just drop this patch for now and first provide the functionality for components and then for the controlplane.

Reply:
This works just fine as discussed on the call, the namespace is updated the first thing after installation of the cluster.